app/javascript/application.js

```diff
@@ -0,0 +1,17 @@
+import 'babel-polyfill'
+import jQuery from "jquery";
+import { initAll } from "govuk-frontend";
```

Comment:
Can we stick or twist with double vs single quotes?

Reply:
I'll to a `standard --fix`